ui/app/AppLayouts/CommunitiesPortal/panels/CommunityTagsPanel.qml

```diff
@@ -96,6 +104,11 @@ StatusScrollView {
             Layout.fillWidth: true
         }
 
+        Item {
+            Layout.preferredHeight: 1
+            Layout.fillWidth: true
+        }
```

Comment:
Why these 1px high separators?

Reply:
There is additional spacing above and below tags picker in mockups, i've replaced it with additional ColumnLayout